lib/helpers.js

```diff
@@ -94,7 +93,12 @@ const formatOnSaveIfEnabled = () => {
     return;
   }
 
-  format(null, { ignoreSelection: true });
+  const formatOnSaveSilentError = getConfigOption('formatOnSaveSilentError');
+
+  format(null, {
+    ignoreSelection: true,
+    ignoreErrors: !formatOnSaveSilentError,
```

Comment:
I would just inline the call to `getConfigOption`, or at least name the variablje `ignoreErrors` so you can just use property shorthand